AUTHORS.md

```diff
@@ -1,13 +0,0 @@
-# Contributors to Acts in alphabetical order:
-
-- Andreas Salzburger, CERN
-- Bastian Schlag, CERN, JGU Mainz
-- Christian Gumpert
-- Fabian Klimpel, Max-Planck-Institut für Physik
-- Hadrien Grasland, Centre National de la Recherche Scientifique
-- Julia Hrdinka
-- Moritz Kiehn, Université de Genève
-- Noemi Calace, CERN
-- Paul Gessinger, CERN, JGU Mainz
-- Robert Langenberg, CERN
-- Xiaocong Ai, LBNL
```

Comment:
Do we not want to have this in the repo root anymore?

Reply:
Do we need an author list if author names are already recorded in git commits?

Reply:
Yes, because unfortunately who's on here can be a political decision. I had the list on the old page be generated from the committers automatically, and had do disable that to remove someone.

Reply:
After discussing this with @paulgessinger I moved the information back into the `AUTHORS` file in the project root. The documentation is including the file automatically.

@HadrienG2 The resulting authors page in the generated documentation extends the file and includes a link to the contributors page on Github.